samples/SunburstChartExample/sunburstppt.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.IO.Packaging" Version="4.7.0" />
```

Comment:
This shouldn't be required

Reply:
I'll remove this after testing with the nuget package (?)

Reply:
I'll remove this after testing with the nuget package (?)

Reply:
removed by referencing documentformat.openxml project.